packages/docusaurus-plugin-google-gtag/src/gtag.ts

```diff
@@ -23,14 +23,10 @@ const clientModule: ClientModule = {
       setTimeout(() => {
         // Always refer to the variable on window in case it gets overridden
         // elsewhere.
-        window.gtag('config', trackingID, {
-          page_path: location.pathname,
-          page_title: document.title,
-        });
```

Comment:
Mmm, why?

Reply:
The config is done [here](https://github.com/facebook/docusaurus/blob/4f26a1911ab140f2a74ffba0122f427c3526261f/packages/docusaurus-plugin-google-gtag/src/index.ts#L72-L73) so doing it again here is redundant.

Reply:
I see—sounds good to me, then!

Reply:
This is actually like a bug fix to me and also something I suspected

When navigating, there were 2 network entries:

<img width="581" alt="CleanShot 2022-06-02 at 12 41 49@2x" src="https://user-images.githubusercontent.com/749374/171612719-61e3185b-1898-452b-96d3-bb887d899c04.png">


Now there's only one 👍